src/eslint/universal.js

```diff
@@ -23,7 +23,6 @@ module.exports = {
   rules: {
     /* Rule definitions and overrides for standard eslint rules */
     camelcase: 'error',
-    'capitalized-comments': ['error', 'always', { ignoreConsecutiveComments: true }],
```

Comment:
I've removed this (despite that I like it) because it is a bit annyoing to work with unless you are used to it... Better not bother at all, since it does not matter much.